devito/interfaces.py

```diff
@@ -89,7 +91,7 @@ class DenseData(SymbolicData):
     therefore do not support time derivatives. Use :class TimeData: for
     time-varying griad data.
     """
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, disk_path=None, **kwargs):
```

Comment:
This is here just because in this implementation, disk_path needs a default value